### ISSUE TITLE
Open URL on a new save active

### ISSUE THREAD

**Issue by GitHubswn** [philc/vimium#3411]
Hello, not sure if this is a bug but when I open a link it opens in a new tab but it doest make it active.

is it just me or its a bug?

to open and activate a url in a new tab, one has to hold down shift key + the letters, correct?

**Comment by innaterebel** [comment]
Not really. It is <kbd>Ctrl</kbd>. <kbd>Shift</kbd> is for to temporarily reverse the link hint mode (open-in-current or open-in-background) you originally invoked.
You can identify what behavior is about to be executed via the HUD located at bottom right of viewport.

By the way, you only need to hold a modifier when you type the last letter.

**Comment by GitHubswn** [comment]
Ok now I'm confused.

The way so far I have understand how to open a link is: Press F letters populate then type the letters desired and it opens in the same tab. 

But if I press Shift or Ctrl or both: the tab just opens in the back ground.  And yes all I see is the HUD "Open link in current tab"

Pls tell step by step how one goes about Opening a tab and make it active.

Thank you.

**Comment by innaterebel** [comment]
Oops, ignored that you are using Safari. I don't think _this_ Vimium project has an official Safari port...?
What's the actual name of the Safari extension you are using?

**Comment by GitHubswn** [comment]
Sorry this supposed to be for Vimari, you guys have almost the same name.


thank you
